upgrading/blue_green_deployments.adoc

```diff
@@ -342,19 +342,21 @@ $ oc delete node --selector=color=blue
 
 .. *Option B* Filter out the masters before running the `delete` command:
 
-... Verify the list of blue node hosts are as expected by running:
+... Verify the list of blue node hosts to delete by running the following
+command. The output of this command includes a list of all node hosts that have 
+the `color=blue` label but do not have the `type=master` label. You can change
```

Comment:
Since the examples from BZ shows that the labels need to exist, I'd propose to change the wording here so that it shows that the value of `type` is different than master, but the label itself must exists, otherwise the command will fail.

Reply:
@soltysh, thanks! I think the section around line 102 might need to be updated a bit, too.

Reply:
@soltysh, I've updated both this section and the section around 102.